website/docs/r/batch_pool.html.markdown

```diff
@@ -247,6 +256,18 @@ A `resource_file` block supports the following:
 
 ~> **Please Note:** Exactly one of `auto_storage_container_name`, `storage_container_url` and `auto_user` must be specified.
 
+---
+
+A `container_registries` block supports the following:
+
+* `registry_server` - (Optional) The container registry URL. The default is "docker.io".
+
+* `user_name` - (Optional) The user name to log into the registry server.
+
+* `password` - (Optional) The password to log into the registry server.
+
+~> **NOTE:** Changes to the password field will not trigger a recreation of the pool, which is necessary to change container registry settings. In order to change the password for a container registry, either taint the Batch Pool and trigger an update, or manually destroy and re-create the pool.
```

Comment:
If we don't read in the password each time updates to password should be reflected?